src/dep_tokenizer.hpp

```diff
@@ -0,0 +1,32 @@
+#ifndef ZIG_DEP_TOKENIZER_HPP
```

Comment:
All this stuff should be in `userland.h`.

Reply:
done. I'm holding off pushing to PR branch until I resolve a windows test failure.